content/millennium/dstu2/individuals/patient.md

```diff
@@ -226,12 +217,6 @@ Content-Type: application/json
 
 The `ETag` response header indicates the current `If-Match` version to use on subsequent updates.
 
-<%= disclaimer %>
-
-### Errors
-
-The common [errors] and [OperationOutcomes] may be returned.
-
 [Time of day of birth]: http://hl7.org/fhir/DSTU2/extension-patient-birthtime.html
```

Comment:
Might want to change these http for https

Reply:
Agreed